hikaricp/tests/fixtures/metrics.txt.old

```diff
@@ -0,0 +1,3 @@
+# HELP hikaricp_connections_timeout_total Connection timeout total count
+# TYPE hikaricp_connections_timeout_total counter
+hikaricp_connections_timeout_total{pool="HikariPool-1",} 0.0
```

Comment:
Not sure this file should be committed, cf its name: `.old`

Reply:
Its a push error, file removed